### DISCUSSION TITLE
2024-08-03-chokuretsu-chinese-chartable

### DISCUSSION THREAD

**Discussion by giscus** [haroohie-club/HaroohieSite#58] [Announcements]
# 2024-08-03-chokuretsu-chinese-chartable

Xzonn explores how he went about modifying the character table and line height for the Chinese translation of Chokuretsu

https://haroohie.club/blog/2024-08-03-chokuretsu-chinese-chartable

**Comment by jonko0493** [comment]
When I first saw this post on Xzonn's blog, it made me so incredibly happy haha. I'm so glad that it's finally made its way to our site. 😊

**Comment by zeez35** [comment]
I didn't understand ANYTHING, but really awesome!!

**Comment by TimepieceMaster** [comment]
Debug logging sounds like something that'd really come in handy! Also fortunate that simply allocating more memory worked lol. With such memory constrained devices like the DS, I've always wondered if many games made use of all the available memory for one reason or another or used memory allocation schemes that forces certain aspects of the game to only use a fixed amount of memory. Like maybe the game decides to reserve a large portion of memory specifically for caching assets that'll be used soon and assumes the font image will only ever be a certain size and so only leaves that much memory available. Thankfully, I haven't heard of people dealing with such restrictions very often, save for the most memory constrained of systems (i.e. 8 and 16-bit systems).
